packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts

```diff
@@ -299,13 +299,18 @@ export class CucumberJSAllureFormatter extends Formatter {
     const currentTest = new AllureTest(this.allureRuntime, Date.now());
     const thread = data.workerId || ALLURE_THREAD_NAME || process.pid.toString();
     const fullName = `${pickle.uri}#${pickle.name}`;
+    // sometimes description starts with "Description:", but it's not actually required
```

Comment:
Cucumber supports i18n. Is it always `Description:` or depends on the used language?

Reply:
True, it doesn't relates on it at all, so we don't need to transform the description

Reply:
Fixed and pushed